src/test/java/com/inventorsoft/junit/service/BookServiceTest.java

```diff
@@ -0,0 +1,230 @@
+package com.inventorsoft.junit.service;
+
+import com.inventorsoft.junit.dto.request.CreateBookRequest;
+import com.inventorsoft.junit.dto.request.UpdateBookRequest;
+import com.inventorsoft.junit.dto.response.AuthorResponse;
+import com.inventorsoft.junit.dto.response.BookResponse;
+import com.inventorsoft.junit.mapper.BookMapper;
+import com.inventorsoft.junit.model.Author;
+import com.inventorsoft.junit.model.Book;
+import com.inventorsoft.junit.repository.AuthorRepository;
+import com.inventorsoft.junit.repository.BookRepository;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
avoid *. You can configure intelijj to not set it automatically
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-imports